packages/api/src/mediaParsers/applicationFormUrlEncoded.ts

```diff
@@ -1,14 +1,13 @@
 import { URLSearchParams } from 'url';
 import { MediaParser } from '../lib/structures/MediaParser';
-import type { ApiRequest } from '../lib/structures/api/ApiRequest';
-import { MimeTypes } from '../lib/utils/MimeTypes';
+import type { MimeType } from '../lib/utils/MimeType';
 
 export class PluginMediaParser extends MediaParser {
 	public constructor(context: MediaParser.LoaderContext) {
-		super(context, { name: MimeTypes.ApplicationFormUrlEncoded });
+		super(context, { name: 'application/x-www-form-urlencoded' satisfies MimeType });
```

Comment:
Is satisfies needed?

Reply:
Yes, because `name` comes from `PieceOptions`.

Reply:
Either way this is a very temporary patch, as I commented in the [`#plugins`](https://discord.com/channels/737141877803057244/737142940777971773/1274035414348468369) channel at Sapphire, we probably should drop media parsers and let users handle the types on their own.